migrations/1658221793203-Deposit.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class Deposit1658221793203 implements MigrationInterface {
+  name = "Deposit1658221793203";
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "deposit" ADD "realizedLpFeePctCapped" numeric NOT NULL DEFAULT '0'`);
```

Comment:
add a new column that stored the capped value of `realizedLpFeePct`. This will help us to avoid calling min(amount * 12%, realizedLpFeePct) for every deposit row when computing rewards.